packages/video_player/video_player/lib/video_player.dart

```diff
@@ -475,6 +481,19 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     await _applyPlayPause();
   }
 
+  /// The video enters in fullscreen (only works for Web).
+  Future<void> toggleFullScreen() async {
+    if (kIsWeb) {
+      if (!value.isFullScreen) {
+        await _applyFullScreen();
```

Comment:
Since these are only used in one place, they don't need to be separated out into helpers.